packages/react/src/file-uploader/FileUploader.tsx

```diff
@@ -0,0 +1,31 @@
+import { FileUpload } from "@ark-ui/react";
+
+export const FileUploader = () => {
+  return (
+    <FileUpload.Root maxFiles={5}>
+      <FileUpload.Label>File Upload</FileUpload.Label>
+      <FileUpload.Dropzone>Drag your file(s) here</FileUpload.Dropzone>
+      <FileUpload.Trigger>Choose file(s)</FileUpload.Trigger>
```

Comment:
these should come as children

so let's just re-expose the `FileUpload.Label`, `FileUpload.Dropzone`, and `FileUpload.Trigger` components - so devs can choose to piece them back together how they want

the re-exposed `FileUpload.Dropzone` should have styles that make it take up the whole parent container etc.

Reply:
Yes, bhaia. I have exposed these components: 
- FileUploadDropzone
- FileUploadLabel
- FileUploadTrigger
- FileUploadContext (to get the information about `acceptedFiles`, `rejectedFiles`)
- FileUpload (the root component)

I kept the design to the minimum. Please let me know if we want to enforce any design from any of these components.